internal/runner/runner.go

```diff
@@ -160,15 +164,21 @@ func (r *Runner) InputWorkerStream() {
 
 	for sc.Scan() {
 		item := strings.TrimSpace(sc.Text())
-
-		hosts := []string{item}
 		if iputil.IsCIDR(item) {
```

Comment:
what about using a switch here:
```go
switch {
case iputil.IsCIDR(item):
...
case asn.IsASN(item):
...
default:
...
}
```
it would drop the need of `continue`

Reply:
replaced ` if condition ` with `switch`